_sass/buttons.scss

```diff
@@ -0,0 +1,19 @@
+button {
+  background-color: #F9F9F9;
+  border: none;
+  border-radius: 2px;
+  box-shadow: 0 0 3px rgba(0, 0, 0, 0.3);
+  padding: 20px 15px;
+  margin: 10px;
+  font-size: 1.25em;
+  color: #5F7D99;
+
+  &:hover {
+    box-shadow: 0 0 8px rgba(0, 0, 0, 0.3);
```

Comment:
Would also add `cursor: pointer;` here to match the other links on the site.

Reply:
Since these aren't navigation links and just buttons, I would lean toward keeping the default cursor.